contrib/babelfishpg_tsql/src/procedures.c

```diff
@@ -2070,3 +2076,151 @@ gen_sp_droprolemember_subcmds(const char *user, const char *member)
 	rewrite_object_refs(stmt);
 	return res;
 }
+
+static void
+ValidateLinkedServerDataSource(char* data_src)
```

Comment:
Do we really need a function for this?
We can use a macro instead. Moreover, we are using it only once so I dont see a point.